src/components/entity/state-info.html

```diff
@@ -61,22 +61,33 @@
   is: 'state-info',
 
   properties: {
-    detailed: {
-      type: Boolean,
-      value: false,
-    },
-
     stateObj: {
       type: Object,
     },
 
     inDialog: {
       type: Boolean,
+      value: false,
     },
   },
 
-  computeStateName: function (stateObj) {
-    return window.hassUtil.computeStateName(stateObj);
+  _getContext: function () {
```

Comment:
@armills is right here, everything should always be configurable based on attributes, never on placement in the DOM. Because all of a sudden you have to start monitoring all DOM actions. Did element get moved etc.